bug: missing return type causes type failure

Hi 👋 
When playing around with the [Default Starter Kit For TypeScript](https://github.com/fastly/compute-starter-kit-typescript), I noticed a type failure prevents startup:

`node_modules/@fastly/js-compute/types/compute.d.ts:13:19 - error TS7010: 'purgeSurrogateKey', which lacks return-type annotation, implicitly has an 'any' return type.`

Sure enough, when looking at compute.d.ts I see:

`export function purgeSurrogateKey(surrogateKey: string, soft?: boolean);`

(Code [here](https://github.com/fastly/js-compute-runtime/blob/9e4764919c678701accd9cef039de5ff2761f7cc/types/compute.d.ts#L13)).

Looking at the native code, I think it's a boolean result, so I'll try send a PR that adds that.

--- a/test-d/compute.test-d.ts
+++ b/test-d/compute.test-d.ts
@@ -1,0 +1,9 @@
+/// <reference path="../types/compute.d.ts" />
+import { vCpuTime, purgeSurrogateKey } from 'fastly:compute';
+import { expectType } from 'tsd';
+
+// Compute
+{
+  expectType<number>(vCpuTime());
+  expectType<boolean>(purgeSurrogateKey('boo'));
+}

--- a/types/compute.d.ts
+++ b/types/compute.d.ts
@@ -10,5 +10,8 @@ declare module 'fastly:compute' {
    * @param soft - Enable to perform a soft purge, retaining stale cache entries to
    *               reduce load on the origin server, defaults to a hard purge.
    */
-  export function purgeSurrogateKey(surrogateKey: string, soft?: boolean);
+  export function purgeSurrogateKey(
+    surrogateKey: string,
+    soft?: boolean,
+  ): boolean;
 }

